README.md

```diff
@@ -1,22 +1,27 @@
-# Unit-Testing-Article-Group4
+# Unit Testing Article 
+Group4 
+
+---
+---
+
```

Comment:
This is probably a better way to present the heading of your article. You should swap out 'Group4' with all your names, however.